docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
+        |-- requirements.txt
+
+If you've not called ``set_tracking_uri`` or set the ``MLFLOW_TRACKING_URI`` environment variable to point to a remote tracking server, this **model** directory will be under the **mlruns** directory.
+
+For more information, including a list of supported model flavors and storing your own flavor, see :ref:`models_built-in-model-flavors`.
+
+Load a model from a specific training run for inference
+--------------------------------------------
+
+To load and run a model stored in a previous run, you can use the ``mlflow.{library_module_name}.load_model`` function. You'll need the run ID of the run that logged the model. You can find the run ID in the tracking UI:
+
+.. image:: _static/images/quickstart/quickstart_run_id.png
+    :width: 400px
+    :align: center
+    :alt: Screenshot showing location of run ID in tracking UI
+
+.. code-section::
 
     .. code-block:: python
 
         import mlflow
 
-        mlflow.set_tracking_uri("databricks")
-        # Note: on Databricks, the experiment name passed to set_experiment must be a valid path
-        # in the workspace, like '/Users/<your-username>/my-experiment'. See
-        # https://docs.databricks.com/user-guide/workspace.html for more info.
-        mlflow.set_experiment("/my-experiment")
-
-    .. code-block:: R
-
-        library(mlflow)
-        install_mlflow()
-        mlflow_set_tracking_uri("databricks")
-        # Note: on Databricks, the experiment name passed to mlflow_set_experiment must be a
-        # valid path in the workspace, like '/Users/<your-username>/my-experiment'.  See
-        # https://docs.databricks.com/user-guide/workspace.html for more info.
-        mlflow_set_experiment("/my-experiment")
+        from sklearn.model_selection import train_test_split
+        from sklearn.datasets import load_diabetes
+
+        db = load_diabetes()
+        X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
+
+        model = mlflow.sklearn.load_model("runs:/97fd7ade5106ee341e0b4c63a53a9776231")
+        predictions = model.predict(X_test)
+        print(predictions)
+
+Note that while ``log_model`` saves environment-specifying files such as **conda.yaml** and **requirements.txt**, ``load_model`` does not automatically recreate that environment. To do so, you need to use your preferred method (**conda**, **virtualenv**, **pip**, etc.), using the artifacts saved by ``log_model``.
+
+To learn more about loading models for specific runs, see :ref:`quickstart_drilldown_log_and_load_model`.
+
+Next steps
+----------
+..
+    First, code:
+
+- :ref:`MLflow tutorials and examples <tutorials-and-examples>`
+- Use the MLflow Registry to store and share versioned models, see :ref:`registry`
+- Use MLflow Projects for packaging your code in a reproducible and reusable way, see :ref:`projects`
+- Use MLflow Recipes (experimental) to create workflows for faster iterations and easier deployment, see :ref:`recipes`
```

Comment:
We can remove "experimental" references for MLflow Recipes. It's been stabilized with the 2.3.0 release

Reply:
DONE